src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -2851,6 +2851,91 @@ static void Main()
             comp.VerifyEmitDiagnostics();
         }
 
+        [Fact]
+        public void TypeInference_Nullable()
```

Comment:
`Nullable` is a bit of an ambiguous term now, let's clarify what nullable we mean :).
```suggestion
        public void TypeInference_NullableValueType()
```